Transfer/Sources/TransferAmountCell.swift

```diff
@@ -100,9 +100,19 @@ final class TransferAmountCell: UITableViewCell {
 
     func configure(amount: String?, currency: String?, isEnabled: Bool, _ handler: @escaping EnteredAmountHandler) {
         titleLabel.text = "transfer_amount".localized()
+        titleLabel.adjustsFontForContentSizeCategory = true
+        titleLabel.numberOfLines = 0
+        titleLabel.font = Theme.Label.captionOne
```

Comment:
This is being set from ThemeManager so remove it from here.
Also update `
@objc dynamic var titleLabelFont: UIFont! { 
   get { return titleLabel.font } 
   set { 
       titleLabel.font = newValue 
       amountTextField.font = newValue 
    } 
}
`
```@objc dynamic var titleLabelColor: UIColor! {
    get { return titleLabel.textColor }
    set {
        titleLabel.textColor = newValue
        amountTextField.textColor = newValue
    }
} ```

Reply:
Changed made